gocwebtemplate-core/gocwebtemplate-core-base/src/main/java/goc/webtemplate/Link.java

```diff
@@ -6,6 +6,7 @@
 public class Link implements Serializable, Cloneable {
     private String href = "";
     private String text = "";
+    private boolean newWindow;

```

Comment:
I like Option 2 a little better as well, as it would not require any changes to existing classes. 

I don't think this change will impact anything on the .NET side of things. The link class in .NET has the newWindow property but I don't think that difference between Java and .NET is a concern.

Reply:
done